src/components/ClusterNodes/NodeList.tsx

```diff
@@ -295,7 +296,19 @@ export default function NodeList({ imageList, isSuperAdmin, namespaceList }: Clu
                 if (searchedTextMap.size !== matchedLabelCount) {
                     continue
                 }
+            } else if (selectedSearchTextType === 'nodeGroup') {
```

Comment:
we should create some constant for nodeGroup as it is being used on multiple places